test/test_runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Get current test file name
+TESTFILE_NAME=${PGAPPNAME##pg_regress/}
+# Set different name for each test database
+# As pg_regress does not support cleaning db after each test
+TEST_CASE_DB="ldb_${TESTFILE_NAME}"
+
+# Drop db after each test on exit signal
+function drop_db {
```

Comment:
`pg_regress` has the option of running tests in parallel. How does that option interact with this dropping logic?

Reply:
Yes currently all the tests are being run in parallel, and each test case is creating a database for it's own with the name like `ldb_hnsw` or `ldb_hnsw_inserts` like this.